### DISCUSSION TITLE
v1.87.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#5114] [Announcements]
# v1.87.0

Welcome to release `v1.87.0` of Immich. This release is the final version where we bid farewell to the `immich-web` and `immich-proxy` containers. Starting in the next release, the two containers will be consolidated into the `immich-server` container for easier deployment and maintenance. For more information on preparing for the next release, please read the following [announcement](https://github.com/immich-app/immich/discussions/5086)

Recently, I've had a chance to get on the **Self-hosted.show** podcast to talk about Immich, the team behind it, the project's future and vision, as well as the sustainability of the project. You can have a listen [here]( https://selfhosted.show/110?t=1198).

Please find the highlights of this release below:

- You can now use the mobile app without giving it full permission to access media storage, allowing you to access only remote assets that are backed up on the server.
- Machine learning models are now unloaded when unused, reducing the application's memory footprint.
- The `/asset/import` endpoint has been removed in favor of Libraries.
- Lower thumbnail resolution options

- - - -

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Server
* feat(server): asset search endpoint by @jrasm91 in https://github.com/immich-app/immich/pull/4931
* chore(server): pretty print json object in error message by @ishanjain28 in https://github.com/immich-app/immich/pull/5036
* chore(server): improve e2e test speed by @jrasm91 in https://github.com/immich-app/immich/pull/5026
* chore(server): cleanup e2e by @jrasm91 in https://github.com/immich-app/immich/pull/5051
* chore(server): shared link e2e for add/remove assets by @jrasm91 in https://github.com/immich-app/immich/pull/5052
* chore(server): remove asset search endpoint by @jrasm91 in https://github.com/immich-app/immich/pull/5094
* chore(server): remove import file endpoint by @jrasm91 in https://github.com/immich-app/immich/pull/5093

### Mobile
* feat(mobile): use app without storage permission by @fyfrey in https://github.com/immich-app/immich/pull/5014
* feat(mobile): Add es-US translations by @adamantike in https://github.com/immich-app/immich/pull/4997
* feat(mobile): remove announcement overlay and show in app bar dialog by @shenlong-tanwen in https://github.com/immich-app/immich/pull/5104
* fix(mobile): partner assets in albums & share by @fyfrey in https://github.com/immich-app/immich/pull/5029
* fix(mobile): rebuild of unified partner timeline by @fyfrey in https://github.com/immich-app/immich/pull/5065
* fix(mobile): update local deleted assets in sync by @fyfrey in https://github.com/immich-app/immich/pull/5099
* chore(mobile): Translations update by @alextran1502 in https://github.com/immich-app/immich/pull/5053

### Web
* fix(web): back button from person detail page by @martabal in https://github.com/immich-app/immich/pull/5047
* fix(web): previous url in person page by @martabal in https://github.com/immich-app/immich/pull/5071
* chore(web): annoucement box for breaking change by @alextran1502 in https://github.com/immich-app/immich/pull/5091
* feat(web): add (lower) thumbnail resolution options  by @davidacampos in https://github.com/immich-app/immich/pull/5107

### Machine Learning
* fix(ml): better model unloading by @mertalev in https://github.com/immich-app/immich/pull/3340

### Others
* chore(test): don't set max old space in e2e tests by @etnoy in https://github.com/immich-app/immich/pull/5050
* chore: readme feature list by @alextran1502 in https://github.com/immich-app/immich/pull/5048
* chore(deps): bump actions/github-script from 6 to 7 by @dependabot in https://github.com/immich-app/immich/pull/5032
* fix(deps): update dependency commander to v11.1.0 by @renovate in https://github.com/immich-app/immich/pull/5003
* chore(deps): update postgres:14-alpine docker digest to 50d9be7 by @renovate in https://github.com/immich-app/immich/pull/5054
* chore(deps): update dependency @faker-js/faker to v8 by @renovate in https://github.com/immich-app/immich/pull/5059
* chore(deps): update dependency @testing-library/jest-dom to v6 by @renovate in https://github.com/immich-app/immich/pull/5073
* chore: added new zcash donation option in readme by @JordyEGNL in https://github.com/immich-app/immich/pull/5087

## New Contributors
* @ishanjain28 made their first contribution in https://github.com/immich-app/immich/pull/5036
* @davidacampos made their first contribution in https://github.com/immich-app/immich/pull/5107

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.86.0...v1.87.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.87.0'>v1.87.0</a>.</em>

**Comment by Nicky5** [comment]
Hi, since importing is no longer supported is there a way to transfer the pictures i previously imported using the cli to a library?
